metric/metric.go

```diff
@@ -0,0 +1,59 @@
+package metric
+
+import (
+	"github.com/fabric8-services/fabric8-common/log"
```

Comment:
Shouldn't we use the fabric-auth log package here?

Reply:
good point, @sbryzak

Reply:
fixed in 5d6a0b4